### ISSUE TITLE
.tojson() gives null as result

### ISSUE THREAD

**Issue by adriancoca** [orientechnologies/orientdb#10264]
## Expected behavior  
Should stringify node content

## Actual behavior  
array with null value [null]

## Steps to reproduce  
Example select

```
select routeDataNodes,routeDataNodes.tojson() as routeDataJSON
FROM ( select * from ( 
  select key,uuid as commandUuid,start_date,out("ON_ROUTE") as routeDataNodes  /*out("ON_ROUTE").tojson() as earlyjson   corect json calculated */
FROM task
WHERE  start_date >= '2024-02-05 00:00:00' and start_date <= '2024-07-02 23:59:59'
) where key="COMANDA" and commandUuid="16821cf6-4ed6-4ba3-9ceb-0654ad4f2277" )

order by start_date desc
```

![image](https://github.com/orientechnologies/orientdb/assets/7643185/0e6cc521-5834-4dca-906d-15c39605c26b)

**Comment by tglman** [comment]
Hi,

I fixed this in the last develop branches ( so also 3.2.x) and should be released with the 3.2.32 hotfix.

with the fix the two way of do toJson give the same result.

Regards

**Comment by tglman** [comment]
Hi,
3.2.32 has been released with the fix, feel free to check if it works ok also at your side
